src/Tooltip.js

```diff
@@ -0,0 +1,130 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import Box from './Box'
+import Card from './Card'
+
+import theme from './theme'
+
+import styled from 'styled-components'
+
+const arrowShadow = props => {
+  return props.top
+    ? {
+        'box-shadow':
+          '-9.66px 9.66px 8px 0 rgba(0,0,0,0.04), -4px 4px 4px 0 rgba(0,0,0,0.08)'
+      }
+    : {
+        'box-shadow':
+          '-1.41px 1.41px 1px 0 rgba(0,0,0,0.01), -3.66px 3.66px 8px 0 rgba(0,0,0,0.04)'
+      }
+}
+
+const arrowAlign = props => {
+  return props.left
+    ? { left: '16px', 'margin-left': props.top ? 0 : '15px' }
+    : props.center
+      ? { left: '50%', 'margin-left': props.top ? '-7px' : '7px' }
+      : { right: '16px', 'margin-right': props.top ? '5px' : '-10px' }
+}
+
+const arrowPosition = props => {
+  return props.top
+    ? {
+        'transform-origin': '0 0',
+        transform: 'rotate(-45deg)',
+        bottom: '-10px'
+      }
+    : {
+        'transform-origin': '0 0',
+        transform: 'rotate(-225deg)',
+        top: '0'
+      }
+}
+
+const arrow = props => {
+  return props.top
+    ? {
+        'transform-origin': '0 0',
+        transform: 'rotate(-45deg)'
+      }
+    : {
+        'transform-origin': '0 0',
+        transform: 'rotate(-225deg)'
+      }
+}
+
+const tooltipPosition = props => {
+  return props.top ? { bottom: '-8px' } : { top: 0 }
+}
+
+const tooltipAlign = props => {
+  return props.right
+    ? { right: 0 }
+    : props.center
+      ? { left: '50%', width: 'auto', transform: 'translateX(-50%)' }
+      : null
+}
+
+const TooltipContent = styled(Box)`
+  display: ${({ isVisible }) => (isVisible ? 'inline' : 'none')};
+  box-shadow: ${({ theme }) => theme.boxShadows.slice(0, 3).join(', ')};
+  font-size: ${({ theme }) => theme.fontSizes[0]}px;
+  position: absolute;
+  border-radius: ${({ theme }) => theme.radii[1]}px;
+  box-sizing: border-box;
+  background: ${({ theme, bg }) => theme.colors[bg]};
+  text-align: center;
+
+  ${tooltipPosition} ${tooltipAlign} &::after {
+    content: '';
+    position: absolute;
+    width: 0;
+    height: 0;
+    border-width: 5px;
+    border-style: solid;
+    border-color: transparent transparent ${({ theme, bg }) => theme.colors[bg]}
+      ${({ theme, bg }) => theme.colors[bg]};
+
+    ${arrow} ${arrowPosition} ${arrowAlign} ${arrowShadow};
+  }
+`
+
+const propTypes = {
+  children: PropTypes.any.isRequired,
+  bg: PropTypes.string,
+  color: PropTypes.string,
+  bottom: PropTypes.bool,
+  top: PropTypes.bool,
+  center: PropTypes.bool,
+  left: PropTypes.bool,
+  right: PropTypes.bool,
+  isVisible: PropTypes.bool
+}
+
+const defaultProps = {
+  position: 'bottom',
+  color: 'text',
+  bg: 'white',
+  align: 'right'
```

Comment:
We've started adding `theme` as a default prop to help with testing – would be good to pass it to the TooltipContent component IMO

Reply:
@jxnblk how come it was recently removed from here >>
<img width="992" alt="screen shot 2017-12-07 at 10 38 59 am" src="https://user-images.githubusercontent.com/1365995/33723508-e61b4442-db3a-11e7-9c2a-b62f1feb5d58.png">

Reply:
@krevat It was remove from icons since we really didn't need theme there. 
@jxnblk I feel like it is a workaround for now but I think we should figure out another way to solve testing components with using the theme. I would still prefer wrapping the component in the test with a `ThemeProvider`.

Reply:
Happy to discuss it again if there's any concerns, but we already discussed passing `theme` as a default prop, and the other components do this as well.
For this PR, I think it would be good to follow conventions with the other components in this library – we also don't define `propTypes` and `defaultProps` at the beginning of the file elsewhere, but that's something we haven't discussed yet.

Reply:
Yes, I am kinda questioning the convention here haha. I think that adding `theme` as a default prop is a good workaround for simple components that doesn't contain styled-components as a children that consumes the `theme` in their styles. If the component has a lot of nested components the `theme` will not exist vs in `ThemeProvider` it will since it's passed through the context.